projects/storefrontlib/cms-components/misc/site-context-selector/language-currency.component.ts

```diff
@@ -1,11 +1,18 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { SiteContextType } from './site-context.model';
 
 @Component({
   selector: 'cx-language-currency-selector',
   template: `
-    <cx-site-context-selector context="language"></cx-site-context-selector>
-    <cx-site-context-selector context="currency"></cx-site-context-selector>
+    <cx-site-context-selector
+      [context]="SiteContextType.LANGUAGE"
+    ></cx-site-context-selector>
+    <cx-site-context-selector
+      [context]="SiteContextType.CURRENCY"
+    ></cx-site-context-selector>
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class LanguageCurrencyComponent {}
+export class LanguageCurrencyComponent {
+  readonly SiteContextType = SiteContextType;
```

Comment:
What's the new attribute for?
It probably needs to start with a lowercase letter.

Reply:
`SiteContextType` is enum, the `[context]` passed to `selector` should be enum. I will rename it lowercase.